extensions/yarn/README.md

```diff
@@ -42,6 +46,39 @@ In cases where Phylum still needs to process some of the packages, the command
 will exit with a warning **without** installing the packages. Once the analysis
 is complete, another attempt can be made.
 
-[phylum]: https://phylum.io
-[phylum-cli]: https://github.com/phylum-dev/cli
-[yarn]: https://yarnpkg.com/
+## Troubleshooting
+
+There are some common reasons this extension may fail.
+
+### Sandbox violations
+
+The underlying package manager may fail due to operating within the Phylum
+sandbox. These errors are usually displayed in the output and say something
+about not being able to access a particular file or network resource.
```

Comment:
This is inaccurate, at least for all extensions. Often times there are no error messages and while network resource errors might be possible, I think we have networking enabled pretty much everywhere?

Reply:
I will attempt to reword this to be more universal. The `yarn` and `npm` extensions *disable* networking during the sandbox building step. The remaining package manager extensions *enable* networking for all `runSandboxed` calls.